### PR TITLE
Add note about switcher needing access to port 20002 UDP

### PR DESCRIPTION
**Pull request by guysoft**
## Proposed change
See PR  #22083 that this PR is a fix of
The changes are based on instructions left in review: https://github.com/home-assistant/home-assistant.io/pull/22083#pullrequestreview-915146848

Had to run for UFW:
```
sudo ufw allow 20002/udp
```

You can see its being used here in the lib this integration imports:
https://github.com/TomerFi/aioswitcher/blob/dev/src/aioswitcher/bridge.py#L112

## Proposed change

I could not understand why my switcher is not detected. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ x ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ x ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ x ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/switcher_kis.markdown [hunk 1]
```diff
@@ -62,3 +62,8 @@ Note: This does not affect the auto off timer.
 | ------------- | --------- | -------------------------------------------------------------------------------------- | -------------------------- |
 | `entity_id`   | Yes       | Name of the entity id associated with the integration, used for permission validation  | switch.switcher_kis_boiler |
 | `timer_minutes`    | Yes       | Integer containing timer minutes (valid range 1 to 150)                                      | 90                    |
+
```
Comment (thecode) [inline source/_integrations/switcher_kis.markdown:65]:
```suggestion
```
Extra empty line

File: source/_integrations/switcher_kis.markdown [hunk 2]
```diff
@@ -62,3 +62,8 @@ Note: This does not affect the auto off timer.
...
+
+## Notes
+
+If Home Assistant and your Switcher are not on the same network, you need to create a firewall rule, which allows a connection on port 20002 with the UDP protocol from Switcher to your TV.
```
Comment (thecode) [inline source/_integrations/switcher_kis.markdown]:
```suggestion
If Home Assistant and your Switcher are not on the same network, you need to create a firewall rule, which allows a connection on port 20002 with the UDP protocol from Switcher to your Home Assistant and port 9957 with the TCP protocol from Home Assistant to your Switcher.
```

**Review by thecode — CHANGES_REQUESTED** [review]

File: source/_integrations/switcher_kis.markdown (hunk 2 above)
Comment (guysoft) [reply source/_integrations/switcher_kis.markdown]: Switcher doesn't really have any configurations to set a firewall.  Its an esp essentially.  I didn't need to set such a setting on the homesssistant firewall

Comment (thecode) [reply source/_integrations/switcher_kis.markdown]: Your installation does not have a firewall for outgoing connections, but some users run devices on a separate IOT VLAN and needs to create firewall rules also for outgoing connections not only for incoming connections.
If we add documentation for connections we need to document all the ports needed. It is up to the user to verify what he needs in his installation.

Comment (guysoft) [reply source/_integrations/switcher_kis.markdown]: Ok, understood. merged

**Review by thecode — APPROVED** [review]
Thanks

**Pull request merged**